config/compat.go

```diff
@@ -97,6 +97,7 @@ type configV112 struct {
 	} `yaml:"rhp3,omitempty"`
 	Log struct {
 		// Path is the directory to store the hostd.log file.
+		//
```

Comment:
Inb4 anyone complains about this madness. It's the updated linter that wants this now. Including getting rid of the `+build` tags.